src/robot/parsing/lexer/blocklexers.py

```diff
@@ -200,7 +201,7 @@ def _handle_name_or_indentation(self, statement):
     def lexer_classes(self):
         return (TestOrKeywordSettingLexer, BreakLexer, ContinueLexer,
                 ForLexer, InlineIfLexer, IfLexer, ReturnLexer, TryLexer,
-                WhileLexer, KeywordCallLexer)
+                WhileLexer, SyntaxErrorLexer, KeywordCallLexer)
```

Comment:
`SyntaxErrorLexer` should handle also `BREAK` and `CONTINUE` to allow removing `BreakLexer` and `ContinueLexer` from here. It probably could also handle `RETURN`, but if `ReturnLexer` is removed here then `KeywordLexer` needed to add it to its `lexer_classes`.